app/remoteCam/video.py

```diff
@@ -14,8 +14,8 @@
 
 if __name__ == "__main__":
 #    irled = led.sensor(3)  # Instantiate led class and assign the pin the BCM3
-
-    of = "/home/pi/Videos/"  # output folder
+    of = "/home/pi/IntermediateVideos/" # output folder
```

Comment:
Avoid variable names like this. It saves a few keystrokes but makes reading the code much harder.

`working_folder` and `final_folder` would be much  better than `of` and `ff`.